### PR TITLE
Remove empty lines in the doc generator  to improve readability

### PR DESCRIPTION
**Pull request by letzya**
## Description

1. Remove empty lines in the doc generator  to improve readability
2. Change the file name to include `Definition` as in **TYK OAS API Definition Object**

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Refactoring or add test (improvements in base code or adds test coverage to functionality)

## Checklist

- [ ] I ensured that the documentation is up to date
- [ ] I explained why this PR updates go.mod in detail with reasoning why it's required
- [ ] I would like a code coverage CI quality gate exception and have explained why

### CODE REVIEW

**Comment by Tyk-ITS** [comment]
**API tests result: skipped** :no_entry_sign: 
Branch used: refs/pull/4660/merge
Commit:  
Triggered by: pull_request (@letzya)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3942189344)

**Comment by Tyk-ITS** [comment]
**API tests result: skipped** :no_entry_sign: 
Branch used: refs/pull/4660/merge
Commit:  
Triggered by: pull_request (@letzya)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3942189344)

File: apidef/oas/middleware.go [hunk 1]
```diff
@@ -100,11 +100,11 @@ type PluginConfig struct {
 	// Driver configures which custom plugin to be used.
 	// It's value should be set to one of the following:
 	//
-	// - `otto`,
-	// - `python`,
-	// - `lua`,
-	// - `grpc`,
-	// - `goplugin`.
+	// - `otto`
```
Comment (titpetric) [inline apidef/oas/middleware.go:103]: We have a style convention for this - commas each line, dot trailing line (godoc-like)

Comment (letzya) [reply apidef/oas/middleware.go]: thanks!!! is there a clear message that I missed? assuming I'm not familiar with tyk repo's ci?

**Comment by letzya** [comment]
my local make lint is fine

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4660/merge
Commit: 8adcd91188aad8c3a464e46e6094702f2e03f5c1 
Triggered by: pull_request (@letzya)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3942356570)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4660/merge
Commit: 150dcedbcd39e8b10a4328f59457bc5b08ae78e2 
Triggered by: pull_request (@letzya)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3948442632)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4660/merge
Commit: 5c476c796961004dc8146546f7b83bd503217112 
Triggered by: pull_request (@letzya)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3952533499)

**Review by jeffy-mathew — APPROVED** [review]

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4660/merge
Commit: 9e8252260baaf5822092c7ebae3fe1fa1f9881c6 
Triggered by: pull_request (@letzya)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3957005387)

**Review by titpetric — APPROVED** [review]

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4660/merge
Commit: 183409bac323013a8521e72b3a8dbd1c99d1c5b0 
Triggered by: pull_request (@letzya)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3958684369)

**Pull request merged**